src/site/content/en/blog/mediastreamtrack-insertable-media-processing/index.md

```diff
@@ -107,17 +105,19 @@ components:
 - The `MediaStreamTrackProcessor`, which consumes a `MediaStreamTrack` object's source and generates a stream
   of media frames, specifically [`VideoFrame`](https://w3c.github.io/webcodecs/#videoframe-interface) or
   [`AudioFrame`](https://w3c.github.io/webcodecs/#audioframe-interface)) objects. You can think of this as a
-  track sink that is capable of exposing the unencoded frames from the track to a `ReadableStream`
-  and that exposes a control channel for signals going in the opposite direction.
+  track sink that is capable of exposing the unencoded frames from the track as a `ReadableStream`.
+  It also exposes a control channel for signals going in the opposite direction.
 - The `MediaStreamTrackGenerator`, which consumes a stream of media frames and exposes a `MediaStreamTrack`
-  interface, so that it can be used in any sink a `MediaStreamTrack` is currently attached to. It takes
-  video frames as input, and emits control signals that result from subsequent processing.
+  interface, so that it can be used in any sink a `MediaStreamTrack` interface.
```

Comment:
copy/pasta or editing issue?

Reply:
Looks like an outdated review comment was used as the basis for a new one. GitHub sometimes does this. PTAL now.